packages/frontend-api/src/Model/Resolver/Products/ProductsResolver.php

```diff
@@ -281,10 +288,25 @@ protected function getOrderingModeFromArgument(Argument $argument): string
     }
 
     /**
+     * @param \Overblog\GraphQLBundle\Definition\Argument $argument
      * @return string
      */
-    protected function getDefaultOrderingMode(): string
+    protected function getDefaultOrderingMode(Argument $argument): string
     {
+        if (isset($argument['search'])) {
+            return ProductListOrderingConfig::ORDER_BY_RELEVANCE;
```

Comment:
Wonder whether this behavior* should be documented somewhere. 
Last paragraph in pagination.md seems like a good place


\* this behavior: default ordering is by priority, but when you add search field, it's relevance

Reply:
**c01fe159b17b299a73d93c4b0ebcb9854a827332**